### ISSUE TITLE
staging deploy failing

### ISSUE THREAD

**Issue by xmunoz** [DARIAEngineering/dcaf_case_management#2783]
Here is what is in papertrail:

```
Dec 19 08:38:41 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535482761289757&selected=1541535482761289757) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535482761289757&selected=1541535482761289757) [4] - Worker 1 (PID: 12) booted in 0.02s, phase: 0
Dec 19 08:38:41 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535482761289758&selected=1541535482761289758) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535482761289758&selected=1541535482761289758) process.pid:12 sqreen.log.level:INFO sqreen.log.location:"/app/log/sqreen.log"
Dec 19 08:38:41 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535482761289759&selected=1541535482761289759) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535482761289759&selected=1541535482761289759) process.pid:10 sqreen.log.level:INFO sqreen.log.location:"/app/log/sqreen.log"
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535482976411648&selected=1541535482976411648) [heroku/web.1](https://my.papertrailapp.com/events?q=program%3Aheroku%2Fweb.1&focus=1541535482976411648&selected=1541535482976411648) State changed from starting to up
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362023&selected=1541535485416362023) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362023&selected=1541535485416362023) E, [2022-12-19T16:38:42.224240 #12.j3s] ERROR -- : Cannot login. Token may be invalid: 
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362026&selected=1541535485416362026) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362026&selected=1541535485416362026) E, [2022-12-19T16:38:42.224403 #12.j3s] ERROR -- : invalid response: {"error"=>"agent type mismatch for app (nodejs) and instance (ruby)", "status"=>false}
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362028&selected=1541535485416362028) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362028&selected=1541535485416362028) E, [2022-12-19T16:38:42.224457 #12.j3s] ERROR -- : Sorry but your Sqreen token appears to be invalid.
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362029&selected=1541535485416362029) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362029&selected=1541535485416362029) Your application is NOT currently protected by Sqreen.
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362032&selected=1541535485416362032) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362032&selected=1541535485416362032) Have you correctly filled your config/sqreen.yml?
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362037&selected=1541535485416362037) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362037&selected=1541535485416362037) E, [2022-12-19T16:38:42.227836 #10.j3s] ERROR -- : Cannot login. Token may be invalid: 
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362039&selected=1541535485416362039) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362039&selected=1541535485416362039) E, [2022-12-19T16:38:42.228003 #10.j3s] ERROR -- : invalid response: {"error"=>"agent type mismatch for app (nodejs) and instance (ruby)", "status"=>false}
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362040&selected=1541535485416362040) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362040&selected=1541535485416362040) E, [2022-12-19T16:38:42.228077 #10.j3s] ERROR -- : Sorry but your Sqreen token appears to be invalid.
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362042&selected=1541535485416362042) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362042&selected=1541535485416362042) Your application is NOT currently protected by Sqreen.
Dec 19 08:38:42 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541535485416362045&selected=1541535485416362045) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541535485416362045&selected=1541535485416362045) Have you correctly filled your config/sqreen.yml?
Dec 19 08:40:49 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541536019594453022&selected=1541536019594453022) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541536019594453022&selected=1541536019594453022) I, [2022-12-19T16:40:49.579726 #12]  INFO -- : [[02c7966c-4494-4820-85c8-1de19af490b0](https://my.papertrailapp.com/systems/daria-services-staging/events?q=%2202c7966c-4494-4820-85c8-1de19af490b0%22&focus=1541536019594453022&selected=1541536019594453022)] Started GET "/users/sign_in" for [172.68.230.151](https://my.papertrailapp.com/systems/daria-services-staging/events?q=%22172.68.230.151%22&focus=1541536019594453022&selected=1541536019594453022) at 2022-12-19 16:40:49 +0000
Dec 19 08:40:49 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541536019594453023&selected=1541536019594453023) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541536019594453023&selected=1541536019594453023) I, [2022-12-19T16:40:49.586580 #12]  INFO -- : [[02c7966c-4494-4820-85c8-1de19af490b0](https://my.papertrailapp.com/systems/daria-services-staging/events?q=%2202c7966c-4494-4820-85c8-1de19af490b0%22&focus=1541536019594453023&selected=1541536019594453023)] Processing by Devise::SessionsController#new as HTML
Dec 19 08:40:49 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541536019594453024&selected=1541536019594453024) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541536019594453024&selected=1541536019594453024) I, [2022-12-19T16:40:49.682743 #12]  INFO -- : [[02c7966c-4494-4820-85c8-1de19af490b0](https://my.papertrailapp.com/systems/daria-services-staging/events?q=%2202c7966c-4494-4820-85c8-1de19af490b0%22&focus=1541536019594453024&selected=1541536019594453024)]   Rendered devise/sessions/new.html.erb within layouts/application (Duration: 24.0ms | Allocations: 8187)
Dec 19 08:40:49 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541536019594453025&selected=1541536019594453025) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541536019594453025&selected=1541536019594453025) I, [2022-12-19T16:40:49.685656 #12]  INFO -- : [[02c7966c-4494-4820-85c8-1de19af490b0](https://my.papertrailapp.com/systems/daria-services-staging/events?q=%2202c7966c-4494-4820-85c8-1de19af490b0%22&focus=1541536019594453025&selected=1541536019594453025)]   Rendered layout layouts/application.html.erb (Duration: 27.0ms | Allocations: 9677)
Dec 19 08:40:49 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541536019594453026&selected=1541536019594453026) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1541536019594453026&selected=1541536019594453026) I, [2022-12-19T16:40:49.686087 #12]  INFO -- : [[02c7966c-4494-4820-85c8-1de19af490b0](https://my.papertrailapp.com/systems/daria-services-staging/events?q=%2202c7966c-4494-4820-85c8-1de19af490b0%22&focus=1541536019594453026&selected=1541536019594453026)] Completed 200 OK in 99ms (Views: 24.9ms | ActiveRecord: 25.7ms | Allocations: 33492)
Dec 19 08:40:50 [daria-services-staging](https://my.papertrailapp.com/systems/daria-services-staging/events?focus=1541536019943825445&selected=1541536019943825445) [heroku/router](https://my.papertrailapp.com/events?q=program%3Aheroku%2Frouter&focus=1541536019943825445&selected=1541536019943825445) at=info method=GET path="/users/sign_in" host=sandbox.dariaservices.com request_id=[02c7966c-4494-4820-85c8-1de19af490b0](https://my.papertrailapp.com/systems/daria-services-staging/events?q=%2202c7966c-4494-4820-85c8-1de19af490b0%22&focus=1541536019943825445&selected=1541536019943825445) fwd="2800:bf0:1c9:103d:98d1:be86:17d7:6c88,[172.68.230.151](https://my.papertrailapp.com/systems/daria-services-staging/events?q=%22172.68.230.151%22&focus=1541536019943825445&selected=1541536019943825445)" dyno=web.1 connect=0ms service=130ms status=200 bytes=4384 protocol=https
```

**Comment by xmunoz** [comment]
Probably not sqreen related, this has been going on since last week: https://github.com/DARIAEngineering/dcaf_case_management/actions/runs/3687046973

**Comment by xmunoz** [comment]
Fixed by #2784
